src/background/content-script.js

```diff
@@ -1,33 +0,0 @@
-import browser from "webextension-polyfill";
-import SLStorage from "../popup/SLStorage";
-
-async function setupContentScript() {
-  const apiUrl = SLStorage.get(SLStorage.SETTINGS.API_URL);
-
-  browser.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
```

Comment:
@nguyenkims this listener was re-registering (as opposed to injecting) the content-script on every tab update. This would create runtime warnings. Solution is declarative content-script in the manifest.